bundles/org.openhab.binding.vizio/README.md

```diff
@@ -0,0 +1,184 @@
+| source      | String    | Select the source input on the TV. The dropdown list is automatically populated from the TV.                                                                                                |
+| activeApp   | String    | A dropdown containing a list of streaming apps defined by the `appListJson` config option that can be launched by the binding. An app started via remote control is automatically selected. |
+| control     | Player    | Control Playback e.g. Play/Pause/Next/Previous/FForward/Rewind                                                                                                                              |
+| button      | String    | Sends a remote control command the TV. See list of available commands below.                                                                                                                |
+
+### List of available button commands for Vizio TVs:
+
+PowerOn  
+PowerOff  
+PowerToggle  
+VolumeUp  
+VolumeDown  
+MuteOn **(may only work as a toggle)**  
+MuteOff **(may only work as a toggle)**  
+MuteToggle  
+ChannelUp  
+ChannelDown  
+PreviousCh  
+InputToggle  
+SeekFwd  
+SeekBack  
+Play  
+Pause  
+Up  
+Down  
+Left  
+Right  
+Ok  
+Back  
+Info  
+Menu  
+Home  
+Exit  
+Smartcast  
+ccToggle  
+PictureMode  
+WideMode  
+WideToggle
+
+### App List Configuration:
+
+The Vizio API to launch and identify currently running apps on the TV is very complex.
+To handle this, the binding maintains a JSON database of applications and their associated metadata in order to populate the `activeApp` dropdown with available apps.  
+
+When the thing is started for the first time, this JSON database is saved into the `appListJson` configuration parameter.
+This list of apps can be edited via the script editor on the thing configuration.
+By editing the JSON, apps that are not desired can be removed from the `activeApp` dropdown and newly discovered apps can be added.
+
+An entry for an application has a `name` element and a `config` element containing `APP_ID`, `NAME_SPACE` and `MESSAGE` (null for most apps):
+
+```
+{
+   "name": "Crackle",
+   "config": {
+      "APP_ID": "5",
+      "NAME_SPACE": 4,
+      "MESSAGE": null
+   }
+},
+
+```
+
+If an app is running that is not currently recognized by the binding, the `activeApp` channel will display a message that contains the `APP_ID` and `NAME_SPACE` which can be used to create the missing record for that app in the JSON.
+
+If an app that is in the JSON database is not recognized by the biding when running or fails to start when selected, try adjusting the `NAME_SPACE` value for that app. `NAME_SPACE` seems to be a version number and adjusting the number up or down may correct the mis-match between the TV and the binding. 
```

Comment:
```suggestion
If an app that is in the JSON database is not recognized by the binding when running or fails to start when selected, try adjusting the `NAME_SPACE` value for that app. `NAME_SPACE` seems to be a version number and adjusting the number up or down may correct the mismatch between the TV and the binding. 
```

Please also put second sentence on a new line, and consider if the first sentence can be simplified. Maybe some commas could solve the complexity, for me it's hard to read "when running or fails to start when selected".